api/proto/flowlog/flowlog.proto

```diff
@@ -0,0 +1,50 @@
+// Copyright(c) 2017-2019 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+option go_package  = "github.com/lf-edge/eve/api/go/flowlog";
+
+option java_package = "com.zededa.cloud.uservice.proto";
+
+message IpFlow {
+  string src = 1;          // Source address
+  int32  srcPort = 2;      // Source port
+  string dest = 3;         // Destination address
+  string destPort = 4;     // Destination port
+  int32  protocol = 5;     // Protocol
+}
+
+message InstanceInfo {
+  string uuid = 1;           // UUID of the instance
+  string intf = 2;           // User visible name
+  string localIntf = 3;      // Internal name of the interface
+  string netInstUUID = 4;    // UUID of the network instance
+}
+
+message FlowRecord {
+  IpFlow flow = 1;
+  int32 aclNum = 2;                         // Matching ACL
+  string action = 3;                        // ACL Action
+  google.protobuf.Timestamp startTime = 4;  // Always non-zero
+  google.protobuf.Timestamp endTime = 5;    // Zero() if flow has not ended
+  int64 txBytes = 6;
+  int64 txPkts = 7;
+  int64 rxBytes = 8;
+  int64 rxPkts = 9;
+}
+
+message DnsRequest {
+  string hostName = 1;                          // Host name
+  repeated string addrs = 2;                    // Ipv4 or Ipv6 address
+  google.protobuf.Timestamp requestTime = 3;    // Time of DNS request
+  int32 aclNum = 4;                             // ACL that resulted in DNS lookup
+}
+
+message FlowMessage {
```

Comment:
Are we adding a new API endpoint/URL for this message?
If so we should document it in API.md.

Reply:
Looking at the current API.md, it seems outdated - still refers to the old "zconfig" and "zmet" etc. Suggest cleaning up API.md and adding the new API as a separate pull request. I can file an issue for that. Agree?

Reply:
OK

Reply:
But please add a comment here for FlowMessage to indicate the name of the new API endpoint.

Reply:
The top level directory lay out is based on API, so, this is a new API. I added the API in the comment.